vector/src/main/java/im/vector/app/features/settings/devices/DevicesViewModel.kt

```diff
@@ -137,12 +137,14 @@ class DevicesViewModel @AssistedInject constructor(
 //                }
 
         session.rx().liveUserCryptoDevices(session.myUserId)
-                .distinct()
-                .throttleLast(5_000, TimeUnit.MILLISECONDS)
+                .map { it.size }
+                .distinctUntilChanged()
+                .throttleFirst(5_000, TimeUnit.MILLISECONDS)
```

Comment:
throttle first is better

Reply:
I think that I made it to wait a bit mainly for the first time. If not just after a new session you could have verification alerts coming to fast

Reply:
Ok, I revert